jkube-kit/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/HelmServiceUtil.java

```diff
@@ -80,6 +82,10 @@ public class HelmServiceUtil {
   protected static final String PROPERTY_HELM_DEBUG = "jkube.helm.debug";
   protected static final String PROPERTY_HELM_DEPENDENCY_VERIFY = "jkube.helm.dependencyVerify";
   protected static final String PROPERTY_HELM_DEPENDENCY_SKIP_REFRESH = "jkube.helm.dependencySkipRefresh";
+  protected static final String PROPERTY_HELM_RELEASE_NAME = "jkube.helm.release.name";
+  protected static final String PROPERTY_HELM_INSTALL_DEPENDENCY_UPDATE = "jkube.helm.install.dependencyUpdate";
+  protected static final String PROPERTY_HELM_INSTALL_WAIT_READY = "jkube.helm.install.waitReady";
+  protected static final String PROPERTY_HELM_INSTALL_DISABLE_OPENAPI_VALIDATION = "jkube.helm.disableOpenAPIValidation";
```

Comment:
This doesn't follow the pattern of other settings.
Is it a generic setting (i.e. it applies to more than one helm goal -install, uninstall, upgrade-)?
Or is it a setting specific to install?

I'd say it's the former, so in this case maybe the field name is not ok.

Reply:
I saw in https://github.com/manusa/helm-java/pull/133 that this field is applicable for install and update, that's why I kept the name as  global one.

I'll rename the field name.